doc/source/features.rst

```diff
@@ -79,6 +79,24 @@ Command line usage will use the new names ::
     positional arguments:
       {friendly_func,func2}
 
+Nested Subcommands
```

Comment:
@anntzer please provide feedback on this -- it is just an initial take.  How do you recommend to document the usage part at the bottom of this section?

Reply:
Something like "the nested subcommands are accessed e.g. by `test.py sub func2`".
Also mention that "subcommands can be nested to an arbitrary level using nested dicts".